packages/simulation-service/simulation-service.ts

```diff
@@ -1 +1,2 @@
-export { ISimulationService as TestInterface } from './src/isimulation';
+export { ISimulationService } from './isimulation';
```

Comment:
I would rather have these files in SRC folder and export them just like that. My bad 😢